webapp/components/_new/generic/UserAvatar/UserAvatar.vue

```diff
@@ -2,18 +2,18 @@
   <div :class="['user-avatar', size && `--${this.size}`]">
     <span class="initials">{{ userInitials }}</span>
     <base-icon v-if="isAnonymous" name="eye-slash" />
-    <img
-      v-else
-      :src="user.avatar | proxyApiUrl"
-      class="image"
-      @error="$event.target.style.display = 'none'"
-    />
+    <responsive-image v-else :image="user.avatar" class="image" sizes="120px" />
```

Comment:
So this is the only place where I decided to set a `sizes`. It's fixed to one size only. I went for 120px which is a little more than the 114px avatar image on the profile page. The avatar image on the post card is 34px wide on my machine. Good @alina-beck ?

If we want to keep that, maybe we want to query for 120px only in the respective graphql request.

Reply:
and giving it `34px, 120px` sizes would do anything because the size it uses is determined by the viewport width, correct? I guess it's not any different on mobile either, as far as I understand how it works.

Reply:
Yes, you the developer must give the browser a hint how wide an image is going to be for a given viewport.